### PR TITLE
v8 folder structure updates

### PR DESCRIPTION
**Pull request by crgrieve**
*Adding new screenshots of folder structure in v8
*Removing references to umbraco_client folder as no longer exists.

Thought I would make this a separate PR as there may be feedback on more items needing updating depending on this.

Thanks,
Carole

### CODE REVIEW

**Comment by kows** [comment]
Shouldn't original version be copied to a "index-v7.md" ?
Version info can be provided by https://our.umbraco.com/documentation/Contribute/adding-metadata .

**Comment by sofietoft** [comment]
Thanks @crgrieve ! 😄 

A few notes for you here:
As @kows also mentions, I think we need to version this article.
`Umbraco_client` is still i Umbraco 7, right?
So, I think it would be great if you would make another article, "index-v7.md". It should contain the same content, except leave the mentions of `Umbraco_Client` in the one for Umbraco 7 💪

**Comment by crgrieve** [comment]
Sure, I will make a v7 version :)

File: Getting-Started/Code/Source-Control/index.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
 versionFrom: 7.0.0
```
Comment (kows) [inline Getting-Started/Code/Source-Control/index.md]: update to 8 ? :)

**Comment by crgrieve** [comment]
@sofietoft , v7 versions added :)

**Comment by sofietoft** [comment]
Brilliant! 😄 Thanks @crgrieve 🎉

**Pull request merged**